gno.land/pkg/gnoland/param_test.go

```diff
@@ -0,0 +1,39 @@
+package gnoland
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParam_Parse(t *testing.T) {
```

Comment:
Unit tests are much cooler in parallel 😎

Reply:
81a3f5651